mne/viz/topomap.py

```diff
@@ -145,13 +145,37 @@ def _add_colorbar(ax, im, cmap, side="right", pad=.05, title=None,
     divider = make_axes_locatable(ax)
     cax = divider.append_axes(side, size=size, pad=pad)
     cbar = plt.colorbar(im, cax=cax, cmap=cmap, format=format)
-    if cmap[1]:
+    if cmap is not None and cmap[1]:
         ax.CB = DraggableColorbar(cbar, im)
     if title is not None:
         cax.set_title(title, y=1.05, fontsize=10)
     return cbar, cax
 
 
+def _remove_zeros(proj):
```

Comment:
This should be renamed to be more informative. How about: "_clean_ssp_data" / "_handle_ssp_format" / "_squeeze_ssp_vectors" or something like that.

Reply:
actually the right name for it should be `_eliminate_zeros` and mimic [scipy's name](https://docs.scipy.org/doc/scipy-0.14.0/reference/generated/scipy.sparse.csr_matrix.eliminate_zeros.html)